desc/vmec.py

```diff
@@ -251,8 +248,8 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         )
 
         ier_flag = file.createVariable("ier_flag", np.int32)
-        ier_flag.long_name = "error flag (0 = solved equilibrium, 1 = unsolved)"
-        ier_flag[:] = int(not eq.solved)
+        ier_flag.long_name = "error flag (1 = error with equilibrium solution)"
```

Comment:
do we want to explain here that we always output 0, so they should check manually?